cumulusci/utils/yaml/cumulusci_yml.py

```diff
@@ -0,0 +1,41 @@
+from typing import IO, Text
+from re import compile, MULTILINE
+from logging import getLogger
+from io import StringIO
+
+import yaml
+
+NBSP = "\u00A0"
+
+pattern = compile(r"^\s*[\u00A0]+\s*", MULTILINE)
```

Comment:
This is a nitpick but I would prefer importing the `re` module and then using `re.compile` here so that it doesn't look like the `compile()` builtin.

Reply:
Fixed.